From be3ad4ee1b3481d31741415efc040cca72de26c2 Mon Sep 17 00:00:00 2001
From: Florian Weimer <fweimer@redhat.com>
Date: Fri, 19 Feb 2016 14:21:34 +0100
Subject: [PATCH] Remove trailing newline from date_fmt in Serbian locales [BZ
 #19581]

(cherry picked from commit ff889b196575c2fbf6aa7130abb1ec862714ea4e)
(cherry picked from commit bbea74b29974d559604691441d12ea80b2abe919)
---
 localedata/ChangeLog           | 7 +++++++
 localedata/locales/sr_ME       | 2 +-
 localedata/locales/sr_RS       | 2 +-
 localedata/locales/sr_RS@latin | 2 +-
 4 files changed, 10 insertions(+), 3 deletions(-)

diff --git a/localedata/ChangeLog b/localedata/ChangeLog
index 05ccd90dd601..dd1c2b102993 100644
--- a/localedata/ChangeLog
+++ b/localedata/ChangeLog
@@ -1,3 +1,10 @@
+2016-02-19  Florian Weimer  <fweimer@redhat.com>
+
+	[BZ #19581]
+	* locales/sr_ME (date_fmt): Remove newline.
+	* locales/sr_RS (date_fmt): Likewise.
+	* locales/sr_RS@latin (date_fmt): Likewise.
+
 2016-02-08  Mike Frysinger  <vapier@gentoo.org>
 
 	* locales/an_ES: Convert to UTF-8 encodings.
diff --git a/localedata/locales/sr_ME b/localedata/locales/sr_ME
index 4f243dc5c6b2..dd68df81f6be 100644
--- a/localedata/locales/sr_ME
+++ b/localedata/locales/sr_ME
@@ -119,7 +119,7 @@ am_pm   "";""
 t_fmt_ampm "<U0025><U0054>"
 date_fmt  "<U0025><U0061><U002c><U0020><U0025><U0065><U002E><U0020>/
 <U0025><U0062><U0020><U0025><U0059><U002E><U0020><U0020><U0025><U0048>/
-<U003A><U0025><U004D><U003A><U0025><U0053><U0020><U0025><U005A><U000A>"
+<U003A><U0025><U004D><U003A><U0025><U0053><U0020><U0025><U005A>"
 week    7;19971130;4
 first_weekday 2
 first_workday 2
diff --git a/localedata/locales/sr_RS b/localedata/locales/sr_RS
index 2ae085b37bc4..ffea86f730f1 100644
--- a/localedata/locales/sr_RS
+++ b/localedata/locales/sr_RS
@@ -300,7 +300,7 @@ am_pm   "";""
 t_fmt_ampm "<U0025><U0054>"
 date_fmt  "<U0025><U0061><U002C><U0020><U0025><U0065><U002E><U0020>/
 <U0025><U0062><U0020><U0025><U0059><U002E><U0020><U0020><U0025><U0048>/
-<U003A><U0025><U004D><U003A><U0025><U0053><U0020><U0025><U005A><U000A>"
+<U003A><U0025><U004D><U003A><U0025><U0053><U0020><U0025><U005A>"
 week    7;19971130;4
 first_weekday 2
 first_workday 2
diff --git a/localedata/locales/sr_RS@latin b/localedata/locales/sr_RS@latin
index da6628b38e30..fd10ea6690b0 100644
--- a/localedata/locales/sr_RS@latin
+++ b/localedata/locales/sr_RS@latin
@@ -120,7 +120,7 @@ am_pm   "";""
 t_fmt_ampm "<U0025><U0054>"
 date_fmt  "<U0025><U0061><U002c><U0020><U0025><U0065><U002E><U0020>/
 <U0025><U0062><U0020><U0025><U0059><U002E><U0020><U0020><U0025><U0048>/
-<U003A><U0025><U004D><U003A><U0025><U0053><U0020><U0025><U005A><U000A>"
+<U003A><U0025><U004D><U003A><U0025><U0053><U0020><U0025><U005A>"
 week    7;19971130;4
 first_weekday 2
 first_workday 2
-- 
2.10.2

